### ISSUE TITLE
Sound check

### ISSUE THREAD

**Issue by carlisia** [vmware-tanzu/thepodlets#72]
### tl;dr for all participants:
- VERY IMPORTANT ‼️: Even after the Zencastr recording stops, PLEASE DO NOT close your recording window until your file has been fully uploaded and you see the message on your screen. If you do so and we lose your track, we lose the entire episode.
- Please use an over-the-ear, **wired** headphones (Bluetooth headphones don't seem to be compatible with Zencastr)
- Mute/unmute on Zencastr only (leave Zoom muted thru the recording)
- Since this is also a video recording, ensure your background is visually ok for perpetuity 😬

## Recommended equipment
- over-the-ear, **wired** headphones (Bluetooth headphones don't seem to be compatible with Zencastr)
- external microphone, preferably dynamic
- a mic stand or a shock mount to avoid external vibrations

If you want to geek out on equipment, see: [Setting Up Your Webcam, Lights, and Audio for Remote Work, Podcasting, Videos, and Streaming](https://mattstauffer.com/blog/setting-up-your-webcam-lights-and-audio-for-remote-work-podcasting-videos-and-streaming/)

## Recording software
- installed https://zoom.us/ for video
- access to Zencastr (over browser)

## During recording
Each episode will have a new Zencastr link. The episode leas will send hosts and guests their individual Zencastr link + a link to the zoom session. Once you have your Zencastr recording link, feel free to click on it in advance so you can check and make sure everything is set-up correctly. Especially inspect the health check.

1) Please plug your headphones into the computer BEFORE you open the Zencastr window - it can be a bit glitchy if you plug headphones in after opening the recording session.
2) Mute/turn off notifications on computer and phone please

### Video instructions
https://vimeo.com/335330315

- Do use Google Chrome or Firefox browsers. Other browsers are not supported.
- Please turn off notifications on your computer, and close your other programs for the duration of the recording.

## Top Dont's for Zencastr recording
- Don't use a mobile device to record.
- Don't run Zencastr through an incognito window or in private browsing mode.
- Don't have extra tabs open on your browser. Close anything you don't need while recording.
- Don't load other computer programs while recording.
